src/Page.js

```diff
@@ -214,8 +214,18 @@ Page.prototype.prepareTemplateData = function () {
     ? this.titlePrefix + (this.title ? TITLE_PREFIX_SEPARATOR + this.title : '')
     : this.title;
 
+  // construct temporary asset object with only POSIX-style paths
+  const asset = {};
+  Object.entries(this.asset).forEach(([key, value]) => {
+    if (key === 'externalScripts') {
```

Comment:
Since `ensurePosix` requires a `string`, perhaps we could use `typeof`

Reply:
As recommended by [`eslint-plugin-lodash/prefer-lodash-typecheck`](https://github.com/wix/eslint-plugin-lodash/blob/master/docs/rules/prefer-lodash-typecheck.md), I have opted for `_.isString(value)`instead.